packages/wallet/src/redux/protocols/indirect-funding/player-a/reducer.ts

```diff
@@ -181,10 +189,41 @@ const waitForApprovalReducer = (
   }
 };
 
+const updateStateWithDirectFundingAction = (
```

Comment:
This function is identical to the function with the same name defined below.

Reply:
I'm not sure I follow, which function is identical to this?